src/Lessons-api.Domain/TeacherModel/AddTeacherDTO.cs

```diff
@@ -1,6 +1,8 @@
-﻿namespace Lessons_api.Domain.TeacherModel
+﻿using Lessons_api.Domain.UserModel;
+
+namespace Lessons_api.Domain.TeacherModel
 {
-    public class AddTeacherDTO : BaseTeacherDTO
+    public class AddTeacherDTO : BaseUserDTO
```

Comment:
Where do you use this model?

Reply:
For response

Reply:
You need to use this as a request body for AddTeacher API

Reply:
When creating a teacher, I only need userId, because it is created based on existing users.